lms/templates/shoppingcart/registration_code_redemption.html

```diff
@@ -24,8 +24,11 @@ <h1 class="title">
             )}" />
       </div>
       <div class="enrollment-details">
-        <div class="sub-title">${_("Confirm your enrollment for:")}
-          <span class="course-date-label">${_("course dates")}</span>
+        <div class="sub-title">
+          ${_("Confirm your enrollment for: {span_start}course dates{span_end}").format(
```

Comment:
@mattdrayer (not sure who to tag here)-- to me, this sentence is really nonsensical. "Confirm your enrollment for: course dates" -- what does that mean? Is that _really_ what this should say?